src/main/resources/TotemCode/TotemMacros.xml

```diff
@@ -311,5 +311,27 @@
 &lt;input type="hidden" id="redirectURL" value="$doc.getURL('view','showEdit=true')"/&gt;
 &lt;input type="hidden" id="showIframe" value="1"/&gt;
 #end
+
+#**
+* Display the totem actions links : Edit totem, documentation ...
+*#
+#macro(displayTotemActionsLinks)
+#set($totemViewURL = $doc.getURL())
+#set($totemEditURL = $doc.getURL('view','showEdit=true'))
+#set($totemDocURL = $xwiki.getURL('TotemCode.Documentation','view'))
+{{html clean="false" wiki="true"}}
```

Comment:
You should avoid this.

* "`clean = false`" so it means if you have errors in your HTML they won't be fixed. I agrea, sometimes we need to disable the cleaner, but here I see no reason to do it.
* "`wiki = true`" means that you can use Wiki syntax inside your macro. This is very dangerous because you can easily have problems by mixing up Wiki and HTML syntaxes. Here, again, I see no reason to do it.